script.js

```diff
@@ -0,0 +1,368 @@
+  keyboard.className = 'keyboard';
+  wrapper.append(keyboard);
+
+  const clearfix = document.createElement('div');
+  clearfix.className = 'clearfix';
+  keyboard.append(clearfix);
+
+  keyboard.after("Shift + alt - change language");
+}
+
+createVirtualKeyboard();
+
+const keyboard = document.querySelector('.keyboard');
+const textarea = document.querySelector('.textarea');
+const clearfix = document.querySelector('.clearfix');
+
+class Button {
+  constructor({ code, key, style }) {
+    this.className = `button ${style}`;
+    this.code = code;
+    this.key = key;
+  }
+
+  buildButton() {
+    const div = document.createElement('div');
+    div.className = this.className;
+    div.dataset.key = this.key;
+    div.dataset.code = this.code;
+    div.innerHTML = this.code;
+    return div;
+  }
+}
+
+function creatButton() {
+  const ar = [];
+  const arr = Object.keys(keyboardElements);
+  arr.forEach((el) => {
+    ar.push(
+      new Button({
+        code: el,
+        key: keyboardElements[el][0],
+        style: keyboardElements[el][0],
+      })
+    );
+  });
+  ar.forEach((el) => {
+    keyboard.append(el.buildButton());
+  });
+}
+
+creatButton();
+showKey();
+textarea.focus();
+
+function showKey() {
+  let arrValues = Object.values(keyboardElements);
+
+  arrValues.forEach((element) => {
+    if (element[0] === 'rShift') {
+      document.querySelector('.rShift').after(clearfix);
+    }
+  });
+
+  if (shiftCase === false && sessionStorage.lang === 'ENG') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[1] !== undefined) {
+        a.innerText = element[1];
+      }
+    });
+  }
+
+  if (shiftCase === true && sessionStorage.lang === 'ENG') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[2] !== undefined) {
+        a.innerText = element[2];
+      }
+    });
+  }
+
+  if (shiftCase === false && sessionStorage.lang === 'RUS') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[3] !== undefined) {
+        a.innerText = element[3];
+      }
+    });
+  }
+
+  if (shiftCase === true && sessionStorage.lang === 'RUS') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[4] !== undefined) {
+        a.innerText = element[4];
+      }
+    });
+  }
+}
+
+keyboard.addEventListener('mousedown', function (event) {
```

Comment:
Better to stick to one code style. If you use arrow functions for callback, so use them everywhere.

Reply:
Fix